packages/create/src/cli/add/new-model.ts

```diff
@@ -0,0 +1,55 @@
+import type { Command } from 'commander';
+
+import { generateChange, ChangeType, getPromptsForNewModel, getVariant } from '@sap-ux/adp-tooling';
+
+import { promptYUIQuestions } from '../../common';
+import { getLogger, traceChanges } from '../../tracing';
+import { validateAdpProject } from '../../validation/validation';
+
+/**
+ * Add a new sub-command to add new odata service and new sapui5 model of an adaptation project to the given command.
+ *
+ * @param {Command} cmd - The command to add the change data-source sub-command to.
```

Comment:
Could you please change the description of the parameter to be related to your command?

Reply:
Renamed in: https://github.com/SAP/open-ux-tools/pull/2098/commits/b05e80600e4d4ba6cc921566e5ceb9ea50e4a996#diff-c7b8e3ff6c0f98d73c8a98d039042c88a52e47bba74dc593d70cf292b0a8cb9eR12